dev/fillInQAHistory.sql

```diff
@@ -0,0 +1,193 @@
+/* 1 - Certification Criteria Added */
+INSERT INTO openchpl.questionable_activity_listing (questionable_activity_trigger_id, listing_id, before_data, after_data, activity_date, activity_user_id, last_modified_user, creation_date, last_modified_date)
+(SELECT (SELECT id FROM openchpl.questionable_activity_trigger WHERE name = 'Certification Criteria Added'), listing_id, null, added_certs, creation_date, activity_user_id, last_modified_user, creation_date, creation_date FROM 
```

Comment:
This is more of question, but I was curious about what the SELECT part returned here so I ran it locally and got one result. I have a fairly recent copy of the database (two-ish weeks old) and so also looked at questionable activity on the dev system to see if there were more criteria added than 1 since it seemed like there probably should be. There are and I guess I'm just wondering if either I ran this incorrectly to only get 1 result or if it's not returning everything; if the latter, did you compare the output of these queries to the questionable activity that is presented in the UI? I would think just do the comparison for a set period of time to try to get a match.